examples/credit-risk-end-to-end/03_Credit_Risk_Model_Training.ipynb

```diff
@@ -0,0 +1,765 @@
+   "id": "69a40de4-65cf-4b45-b321-2b7ce571f8cb",
+   "metadata": {},
+   "source": [
+    "# Credit Risk Model Training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe641d83-1e28-4f7f-895c-8ca038f6cc53",
+   "metadata": {},
+   "source": [
+    "### Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f04f635-401b-47b6-b807-df61d42ec752",
+   "metadata": {},
+   "source": [
+    "AI models have played a central role in modern credit risk assessment systems. In this example, we develop a credit risk model to predict whether a future loan will be good or bad, given some context data (presumably supplied from the loan application process). We use the modeling process to demonstrate how Feast can be used to facilitate the serving of data for training and inference use-cases.\n",
+    "\n",
+    "In this notebook, we train our AI model. We will use the popular scikit-learn library (sklearn) to train a RandomForestClassifier, as this is a relatively easy choice for a baseline model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a96bf1aa-c450-4201-83a4-e25b08bdd12d",
+   "metadata": {},
+   "source": [
+    "### Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a47b33bc-bc06-4de0-8f3a-beea8179035c",
+   "metadata": {},
+   "source": [
+    "*The following code assumes that you have read the example README.md file, and that you have setup an environment where the code can be run. Please make sure you have addressed the prerequisite needs.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c66a3dab-fdbf-40be-8227-6180dc314a84",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Imports\n",
+    "import warnings\n",
+    "import datetime\n",
+    "import feast\n",
+    "import joblib\n",
+    "import pandas as pd\n",
+    "\n",
+    "from feast import FeatureStore, RepoConfig\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.metrics import classification_report"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "2a841445-fa47-4826-a874-28ac0e4ea57f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Ignore warnings\n",
+    "warnings.filterwarnings(action=\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "23579727-7797-4101-a70d-b0d4c24b0fdf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Random seed\n",
+    "SEED = 142"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc5be519-7733-449b-8dc3-411e86371315",
+   "metadata": {},
+   "source": [
+    "This notebook assumes that you have already done the following:\n",
+    "\n",
+    "1. Run the [01_Credit_Risk_Data_Prep.ipynb](01_Credit_Risk_Data_Prep.ipynb) notebook to prepare the data.\n",
+    "2. Run the [02_Deploying_the_Feature_Stores.ipynb](02_Deploying_the_Feature_Stores.ipynb) notebook to configure the feature stores and launch the feature store servers.\n",
```

Comment:
for some reason this link seems to be not working for me. If it works for you ignore my comment.

Reply:
Good catch. I changed the name slightly and this needed to be updated.

Reply:
Fixed.